integrations/pinecone/tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 from haystack_integrations.document_stores.pinecone import PineconeDocumentStore
 
 # This is the approximate time it takes for the documents to be available
-SLEEP_TIME = 10
+SLEEP_TIME = 15
```

Comment:
unrelated. Tests were not stable, so Increased this value.

Reply:
Is that in seconds? Let's add the unit to the comment (or, even better, the identifier).

Reply:
done.